src/Symfony/Component/Serializer/Serializer.php

```diff
@@ -49,7 +49,7 @@ class Serializer implements SerializerInterface, ContextAwareNormalizerInterface
 {
     /**
      * Flag to control whether an empty array should be transformed to an
-     * object (in JSON: {}) or to a map (in JSON: []).
+     * object (in JSON: {}) or to a list (in JSON: []).
```

Comment:
I think I messed up with git because I'm pretty sure to have already fixed this typo :)

Reply:
Yeah, I saw the comments in your PR yesterday.

Reply:
OK, got it. I fixed another comment, but not this one.